xhr/abort-after-stop.htm

```diff
@@ -20,8 +20,11 @@
         client.open("GET", "resources/delay.py?ms=3000", true);
         client.send(null);
         test.step_timeout(() => {
-          assert_equals(abortFired, true);
+          test.step(t => assert_equals(abortFired, true));
           test.done();
+
+          // Prevent harness timeout caused by window.stop() preventing "load" event
+          window.dispatchEvent(new Event('load'));
```

Comment:
Does this emulate some code in the harness that dispatches a load event? How does this work?

cc @jgraham

Reply:
Exactly.  Normally, window.onload is fired, which triggers [`test_environment.add_on_loaded_callback`](https://github.com/w3c/web-platform-tests/blob/master/resources/testharness.js#L1869) to be called.  `window.stop()` prevents that being called, so we invoke it manually

Reply:
Thanks. I think @jgraham needs to sign off on this. It seems we might want dedicated API instead so we can identify the tests that need this more easily.